### ISSUE TITLE
[toc2mp3] The title name will be lost

### ISSUE THREAD

**Issue by Test-224** [cdrdao/cdrdao#6]
If the title name contains multibyte characters, non-ASCII characters (e.g. Japanese) are removed by clean_string().

example:
```
TITLE "(\343\201\202\343\201\204\343\201\206)_\343\201\210\343\201\212? " // (あいう)_えお?
→ "() ?"
```

I want to exclude only control characters or switch for this function.

**Comment by Test-224** [comment]
Thank you for your reply.

Here is the TOC file.
In this file, TITLE is encoded with UTF-8.

[SOLID STATE SURVIVOR.toc](https://github.com/cdrdao/cdrdao/files/8517757/SOLID.STATE.SURVIVOR.toc.txt)

**Comment by Test-224** [comment]
I used "cdrdao read-cddb" to populate it.

Here is my configuration file:
```
cddb_timeout: 60
cddb_server_list: "freedbtest.dyndns.org:/~cddb/cddb.cgi"
```

freedbtest.dyndns.org supports EUC-JP and UTF-8 encoding.

I'm sorry, I don't have any CDs at the moment. But, I don't think most CDs support the UTF-8 encoding of CDTXT.

**Comment by denisleroy** [comment]
toc2mp3 fixed in 1.2.5 with correct re-encoding in both filenames and id3v2 flags.
Burning a TOC with UTF-8 cdtext onto a disc though will yield undefined result. cdrdao will just burn the data unmodified.
